tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -388,11 +394,12 @@ private static Node inlineScript(Node node) throws IOException {
       String code = new String(Files.readAllBytes(getWebfile(href)), UTF_8);
       code = code.replace("</script>", "</JAVA_SCRIIIIPT/>");
       code = INLINE_SOURCE_MAP_PATTERN.matcher(code).replaceAll("");
-      result = replaceNode(
-          node,
-          new Element(Tag.valueOf("script"), node.baseUri(), node.attributes())
-              .appendChild(new DataNode(code, node.baseUri()))
-              .removeAttr("src"));
+      result =
```

Comment:
Perhaps I’m misreading this, but it looks like this diff hunk (and the
ones above, and some below) are no-ops; mind reverting from this commit?
I’d be happy to approve them separately, but it’s hard to tell what
changes are important here.

Reply:
Sorry, I used a formatter (internal; should have applied it to diff only) and it made more changes than I intended. Reverted changes.

Reply:
Thanks! Looks like there are a few more remaining (lines 247, 519, 820).